breaking-front/src/pages/PostWrite/units/PostWriteUploadForm.styles.js

```diff
@@ -0,0 +1,99 @@
+import { ScrollBarX } from 'components/ScrollBar/ScrollBar';
+import styled from 'styled-components';
+
+export const Upload = styled.div`
+  padding: 10px;
+  background-color: ${({ theme }) => theme.gray[50]};
+`;
+
+export const UploadTitle = styled.h3`
+  margin-bottom: 40px;
+  font-size: 24px;
+  text-align: center;
+`;
+
+export const UploadForm = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+export const UploadFileLayOut = styled.div`
+  padding: 10px;
+`;
```

Comment:
```suggestion
export const UploadFileLayout = styled.div`
  padding: 10px;
`;
```

LayOut 으로 되어있는 곳 많은데 Layout으로 수정해주세요